app/forms/nsm/send_back_form.rb

```diff
@@ -49,10 +49,12 @@ def add_further_information_data
       claim.data['resubmission_deadline'] = WorkingDayService.call(working_days_allowed)
       claim.data['further_information'] ||= []
       claim.data['further_information'] << {
+        documents: [],
         caseworker_id: current_user.id,
         information_requested: send_back_comment,
         requested_at: DateTime.current
       }
+      claim.data.merge!('updates_needed' => ['further_information'])
```

Comment:
This shouldn't be needed - it's only relevant for PA because in there there are two sorts of updates that are needed - further information and corrections. NSM has no concepts of correction, so we always know that any updates needed are _always_ further information.

Reply:
@patrick-laa, @robinsk1  is this not temporary? I thought that in the future we might have corrections as well as further info. I think maybe we check with UX and BA.

Reply:
@robinsk1 makes sense as I guess none of the other PRs have included it. Can you make sure you comment on the caseworker overall QA ticket explaining we haven't included that?